src/ui/components/ShowMoreCard/ShowMoreCard.scss

```diff
@@ -2,7 +2,9 @@
 
 .ShowMoreCard-contents {
   &::after {
-    background: linear-gradient(transparent, $base-color);
+    // rgba(255,255,255,0) instead of transparent prevents
+    // grey line in Safari see #2865
```

Comment:
Also I'd put a full-stop after "Safari" and make "See: https://github.com[...]" its own sentence.